### ISSUE TITLE
Missing invocation of response crashes new sandbox

### ISSUE THREAD

**Issue by ryanblock** [architect/architect#197]
**Describe the bug**
If no invocation of response is made, the new sandbox crashes hard. (Not really a boog per se, but could be improved – ideally sandbox never crashes, and errors are simply bubbled to the user.)

Example code:
```
let arc = require('@architect/functions')

function route(req, res) {
  if (1 === 0) {
    res({
      html: 'hi'
    })
  }
}

exports.handler = arc.http(route)
```

Discussed a 3s timeout on awaiting response locally from sandbox before erroring (but not crashing hard).

**Comment by brianleroux** [comment]
added experimental timeout support in 4.3.2 (`npm i @architect/architect@beta`) 

and a test repo on this branch here https://github.com/arc-repos/arc-example-working-locally/tree/test-timeout

**Comment by brianleroux** [comment]
nice catch, try again w `4.3.4` (?)

**Comment by brianleroux** [comment]
published in `4.3.5`
